app/controllers/users_controller.rb

```diff
@@ -1,7 +1,66 @@
 class UsersController < ApplicationController
 
-  def list
+  def show
     @users = User.all
   end
-  alias :index :list
+  alias :index :show
+
+  def new
+  end
+
+  def create
+    @user = User.create(user_params)
+    redirect_to '/'
+  end
+
+  def trim
+    user = User.find(params[:id])
+    user.update_attribute(:trimmed, true)
+    user.save
+    render :json => user
+  end
+
+  def massage_and_trim
+    user = User.find(params[:id])
+    user.update_attributes(:trimmed => true, :massaged => true)
```

Comment:
These days when using ruby 1.9 and above, the rocket arrow syntax is disappearing.  Single colon syntax is much preferred.
```user.update_attributes trimmed: true, massaged: true```

also ... parentheses around a single line function call is unnecessary.

http://effectif.com/ruby/update-your-project-for-ruby-19-hash-syntax